containers/front-end/src/app/heatmap/heatmap.component.ts

```diff
@@ -8,16 +8,24 @@ import {setInterval} from 'timers';
 })
 export class HeatmapComponent implements OnInit {
 
-  private heatMapInterval: any;
+  private HEATMAP_INTERVAL = 1100;
```

Comment:
Change HeatMap interval to 100 and Degraded interval to 100. 
They should be the same, as new grid point comes in, the last grid point gets dropped out (queue system)
Queue length will stay at 500 of the most current
Shorter interval (100) means faster action